debian/postrm

```diff
@@ -28,27 +28,27 @@ case "$1" in
   purge)
     echo PURGE | debconf-communicate $NAME > /dev/null 2>&1 || true
 
-    if [[ -x "/etc/init.d/emby-server" ]] || [[ -e "/etc/init/emby-server.connf" ]]; then
-      update-rc.d emby-server remove >/dev/null 2>&1 || true
+    if [[ -x "/etc/init.d/jellyfin" ]] || [[ -e "/etc/init/jellyfin.connf" ]]; then
+      update-rc.d jellyfin remove >/dev/null 2>&1 || true
     fi
 
     if [[ -x "/usr/bin/deb-systemd-helper" ]]; then
-      deb-systemd-helper purge emby-server.service >/dev/null
-      deb-systemd-helper unmask emby-server.service >/dev/null
+      deb-systemd-helper purge jellyfin.service >/dev/null
+      deb-systemd-helper unmask jellyfin.service >/dev/null
     fi
 
-    userdel emby > /dev/null 2>&1 || true
-    delgroup --quiet emby > /dev/null 2>&1 || true
+    userdel jellyfin > /dev/null 2>&1 || true
+    delgroup --quiet jellyfin > /dev/null 2>&1 || true
     if [[ -d $PROGRAMDATA ]]; then
       rm -rf $PROGRAMDATA
     fi
-    [[ -f /usr/bin/emby-server ]] && rm /usr/bin/emby-server
-    [[ -f /etc/sudoers.d/emby ]] && rm /etc/sudoers.d/emby
-    [[ -d /var/lib/emby-server ]] && rm -rf /var/lib/emby-server
+    [[ -f /usr/bin/jellyfin ]] && rm /usr/bin/jellyfin
+    [[ -f /etc/sudoers.d/jellyfin-sudoers ]] && rm /etc/sudoers.d/jellyfin-sudoers
+    [[ -d /var/lib/jellyfin ]] && rm -rf /var/lib/jellyfin
```

Comment:
I think you forgot to remove the symlink to `/var/lib/emby` here

Reply:
Whoops I do indeed, will add that in an upcoming PR.